packages/services/src/types/types.ts

```diff
@@ -0,0 +1,8 @@
+import { CredentialPayload } from '@vckit/core-types';
+
+export interface IssueEvent {
+  credentialPayload: CredentialPayload;
+  credentialSubject: any;
+  restOfVC: any;
+  context: string[];
+}
```

Comment:
this interface look the same IVcKitIssueVC https://github.com/uncefact/tests-untp/pull/7/files#diff-c027e5b988fa430c7172d5d0e5853b55eef18c83276399b385400f3748329138R5-R7